django/src/rdwatch/tasks/__init__.py

```diff
@@ -96,7 +97,15 @@ def fetch_boundbox_image(
     worldView=False,
 ):
     timebuffer = timedelta(days=1)
-    captures = get_range_captures(bbox, timestamp, constellation, timebuffer, worldView)
+    try:
+        captures = get_range_captures(
+            bbox, timestamp, constellation, timebuffer, worldView
+        )
+    except URLError as e:
+        logger.warning('Failed to get range capture because of URLError')
+        logger.warning(e)
+        return None
```

Comment:
What circumstances would cause a URLError? And should we be setting the task status to "ERROR" accordingly?

Reply:
I'm not entirely sure, it's not a reproducible thing, it was happening last night so I wanted to account for it.  I may be tempted to through a full catch in there and log error for anything that may happen between searching/downloading S3 images.

Reply:
agreed